packages/components/src/Form/Inputs/InputFilters/InputFiltersChip.tsx

```diff
@@ -32,13 +32,21 @@ interface InputFiltersChipProps
   filter: FieldFilter
   onDelete: (field: FieldFilter) => void
 }
+
 export const InputFiltersChip: FC<InputFiltersChipProps> = forwardRef(
   ({ filter, onDelete, ...props }, ref: Ref<HTMLSpanElement>) => {
     const handleDelete = () => onDelete(filter)
     return (
-      <Chip ref={ref} {...props} prefix={filter.field} onDelete={handleDelete}>
-        {filter.value}
-      </Chip>
+      <div>
+        <Chip
+          onDelete={handleDelete}
+          prefix={filter.field}
+          ref={ref}
+          {...props}
+        >
+          {filter.value}
+        </Chip>
+      </div>
```

Comment:
I think it should be no problem to remove `max-width` over on `Chip.tsx` and then you can avoid the extra `div` here, which would be good.

```suggestion
        <Chip
          onDelete={handleDelete}
          prefix={filter.field}
          ref={ref}
          {...props}
        >
          {filter.value}
        </Chip>
```